gm4_soul_fragments/data/gm4_soul_fragments/functions/soul_fragment/dissipate.mcfunction

```diff
@@ -0,0 +1,6 @@
+#@s = soul fragment that is dissipating (@e[type=minecraft:area_effect_cloud,tag=gm4_soul_fragment] at @s)
+#from soul_fragment/sustain
+
+particle minecraft:soul ~ ~.5 ~ .7 0 .7 .01 5 force
+playsound minecraft:block.beacon.ambient master @a[distance=..32] ~ ~ ~ .5 2
```

Comment:
Don't play on `master`. Playing a sound to all players within 32 blocks doesn't do anything when the volume is set to 0.5 (8 blocks).
Either up the volume, add a min volume or decrease the distance requirement. The later is recommended.